src/BTVNanoCommissioning/workflows/ttsemilep_valid_sf.py

```diff
@@ -58,7 +61,126 @@ def accumulator(self):
     def process(self, events):
         isRealData = not hasattr(events, "genWeight")
         dataset = events.metadata["dataset"]
+
+        attributes = dir(events.Jet)
+
+        # Filter attributes that contain 'btagDeepFlav' in their name
+        btag_attributes = [attr for attr in attributes if "btagDeepFlav" in attr]
+
+        # Print the attribute names
+        for attr in btag_attributes:
+            print(attr)
+
+        (
+            print("ProbB present in dataset")
+            if hasattr(events.Jet, "btagDeepFlavB")
+            else None
+        )
+        (
+            print("ProbC present in dataset")
+            if hasattr(events.Jet, "btagDeepFlavC")
+            else None
+        )
+        (
+            print("ProbB is: ", events.Jet.btagDeepFlavB)
+            if hasattr(events.Jet, "btagDeepFlavB")
+            else None
+        )
+        (
+            print(
+                "ProbB calculated is: ",
+                events.Jet.btagDeepFlavB_b
+                + events.Jet.btagDeepFlavB_bb
+                + events.Jet.btagDeepFlavB_lepb,
+            )
+            if hasattr(events.Jet, "btagDeepFlavB_b")
+            and hasattr(events.Jet, "btagDeepFlavB_bb")
+            and hasattr(events.Jet, "btagDeepFlavB_lepb")
+            else None
+        )
+        (
+            print("ProbC is: ", events.Jet.btagDeepFlavC)
+            if hasattr(events.Jet, "btagDeepFlavC")
+            else None
+        )
+        (
+            print(
+                "ProbC calculated is: ",
+                events.Jet.btagDeepFlavCvL
+                / (1.0 - events.Jet.btagDeepFlavCvL)
+                * (events.Jet.btagDeepFlavG + events.Jet.btagDeepFlavUDS),
+            )
+            if hasattr(events.Jet, "btagDeepFlavCvL")
+            and hasattr(events.Jet, "btagDeepFlavUDS")
+            and hasattr(events.Jet, "btagDeepFlavG")
+            else None
+        )
```

Comment:
please remove all the print statements here

Reply:
Removed